src/test/kotlin/com/hibob/academy/dao/PetDaoTest.kt

```diff
@@ -95,11 +145,16 @@ class PetDaoTest @Autowired constructor (private val sql: DSLContext) {
         ownerDao.insertOwner("Bob1", companyId, "123")
 
         val ownerId = ownerDao.getOwnerId(companyId, "123")
```

Comment:
Not sure I understand this 
You add an owner called Bob1 and get it id  as owner id
What is this `val ownerId = ownerDao.getOwnerId(companyId, "123")`
Is it the same as the id we now return on pet? maybe it would be better to change that insertOwner will return its id

Reply:
fixed (it returns owner id (- "ownerDao.getOwnerId"), missed to fix it after I ficed pedDao, I use it to compare owner insertion)